arm-sql/2015-05-01-preview/swagger/automaticTuning.json

```diff
@@ -176,7 +159,7 @@
             "description": "The requested automatic tuning resource state.",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/AutomaticTuningUpdate"
+              "$ref": "#/definitions/AutomaticTuning"
```

Comment:
good to know, only APIs deployed to ARM should be included in a merged spec, if you'd like to have the spec approved and merged for the APIs that are already deployed, you can split this one into another PR. If you expect this API will be deployed soon, we can hold the PR a bit, until it's deployed, let me know. thanks!